docs/dev/table/connectors/formats/orc.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Orc Format"
+nav-title: Orc
+nav-parent_id: sql-formats
+nav-pos: 6
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<span class="label label-info">Format: Serialization Schema</span>
+<span class="label label-info">Format: Deserialization Schema</span>
+
+* This will be replaced by the TOC
+{:toc}
+
+The [Apache Orc](https://orc.apache.org/) format allows to read and write Orc data.
+
+Dependencies
+------------
+
+In order to setup the Orc format, the following table provides dependency information for both
+projects using a build automation tool (such as Maven or SBT) and SQL Client with SQL JAR bundles.
+
+| Maven dependency   | SQL Client JAR         |
+| :----------------- | :----------------------|
+| `flink-orc`        |{% if site.is_stable %} [Download](https://repo.maven.apache.org/maven2/org/apache/flink/flink-orc{{site.scala_version_suffix}}/{{site.version}}/flink-orc{{site.scala_version_suffix}}-{{site.version}}-jar-with-dependencies.jar) {% else %} Only available for stable releases {% endif %}|
```

Comment:
I think it's better to provide the full coordinates of the maven artifact, e.g. `org.apache.flink:flink-orc...`. Or maybe something like
```
		<dependency>
			<groupId>org.apache.flink</groupId>
			<artifactId>flink-orc_{{site.scala_version_suffix}}</artifactId>
			<version>{{site.version}}</version>
		</dependency>
```
Then users can simply copy it to their pom files.

Reply:
It is currently Flink dependency style.

Reply:
But I think we can add `_{{site.scala_version_suffix}}`